### PR TITLE
impl flow mode state subscribe and screen mode cleanup

### PR DESCRIPTION
**Pull request by xavluiz**
This subscribes to the "flow_state" event to update the flow mode editor state if another editor or window has initiated or paused flow

### CODE REVIEW

File: src/managers/FlowManager.ts [hunk 1]
```diff
@@ -55,25 +28,14 @@ export async function enableFlow({ automated = false, skipSlackCheck = false })
 
     (progress) => {
       return new Promise((resolve, reject) => {
-        initiateFlow({ automated, skipSlackCheck }).catch((e) => {});
+        initiateFlow({ automated, skipSlackCheck, process_flow_session }).catch((e) => {});
```
Comment (bjacobson26) [inline src/managers/FlowManager.ts:31]: want to `console.error(e)` ?

File: src/message_handlers/flow_state.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { enableFlow, isFlowModeEnabled, pauseFlow } from "../managers/FlowManager";
+
+export async function handleFlowStateMessage(body: any) {
+  // body contains {enable_flow: true | false}
+  const {enable_flow} = body;
```
Comment (bjacobson26) [inline src/message_handlers/flow_state.ts:5]: nitpick: `const { enable_flow } = body;`

**Review by bjacobson26 — APPROVED** [review]

File: src/managers/FlowManager.ts (hunk 1 above)
Comment (xavluiz) [reply src/managers/FlowManager.ts:31]: good catch. i'll add a code time error message.

File: src/message_handlers/flow_state.ts (hunk 2 above)
Comment (xavluiz) [reply src/message_handlers/flow_state.ts:5]: sounds good. i'll update that and thanks for pointing this out. it helps me format things based on our standards across the various editors.

**Pull request merged**